raw-data/UW/uw_inventories.tsv

```diff
@@ -53,3 +53,43 @@ roseslabrada2015mako.pdf		Mako		i		i
 						n	"/l/ can also be variably nasalized when followed by nasal vowels"	
 				w		w		
 				ˀw		ˀw		
+
+berry1999abun.pdf	abun1252	Abun		p		p		
```

Comment:
Given what the author says about where the data were collected (p3 section 1.4, and p8-9 section 1.7) I think it makes most sense to treat this as a description of Abun Ye dialect. Please add that to the "specific dialect" column, and with that in mind, you can remove notes and allphones that refer to other dialects.

Note that in principle we could consider creating four separate inventory entries (one for each dialect), all based on this one document, if the description of each was sufficiently thorough.  But in this case they explicitly say where their data came from (both villages are within the Abun Ye territory)

Reply:
Understood.